Store/StoreAnalyticsOrderTracker.php

```diff
@@ -31,8 +36,9 @@ class StoreAnalyticsOrderTracker
 	// }}}
 	// {{{ public function __construct()
 
-	public function __construct(StoreOrder $order, $affiliation = null)
+	public function __construct(SiteApplication $app, StoreOrder $order, $affiliation = null)
```

Comment:
Yep. It's needed because CourseHost needs access to the application when disseminating the order. See https://github.com/silverorange/course-host/pull/535/files#diff-72d3b734d3dc38d1d3c19eecc130712aR62 for where it's used.

If there's a better way of doing this, I'm all ears.